src/workflows/recipes/pages/RecipeDetailPage.js

```diff
@@ -49,9 +49,7 @@ class RecipeDetailPage extends React.PureComponent {
               <Col span={4}>
                 <ShieldIdenticon className="detail-icon" seed={revision.get('identicon_seed')} />
               </Col>
-              <Col span={20}>
-                <DetailsActionBar />
-              </Col>
+              <Col span={20}>{revisionId ? <DetailsActionBar /> : null}</Col>
```

Comment:
Nit: Another way to write this is `{revisionId && <DetailsActionBar />}`. I personally think it's cleaner than the `: null` at the end, but I'd r+ either one.

Reply:
I agree. It's an old habit. Don't remember what ancient jsx muscle-memory imprint I have to still not shake this.